src/backend/expungeservice/expunger/analyzers/type_analyzer.py

```diff
@@ -0,0 +1,70 @@
+class TypeAnalyzer:
+
+    def __init__(self):
+        self.class_b_felonies = []
+
+    def evaluate(self, charges):
+        for charge in charges:
+            self.__evaluate(charge)
+
+    def __evaluate(self, charge):
+        if charge.acquitted():
+            charge.expungement_result.set_type_eligibility(True)
+            charge.expungement_result.set_reason('Eligible under 137.225(1)(b)')
+
+        elif TypeAnalyzer.__marijuana_ineligible(charge):
+            charge.expungement_result.set_type_eligibility(False)
+            charge.expungement_result.set_reason('Ineligible under 137.226')
+
+        elif TypeAnalyzer.__list_b_charge(charge):
+            charge.expungement_result.set_reason('Further Analysis Needed')
+
+        elif TypeAnalyzer.__ineligible_under_137_225_5(charge):
+            charge.expungement_result.set_type_eligibility(False)
+            charge.expungement_result.set_reason('Ineligible under 137.225(5)')
+
+        elif TypeAnalyzer.__possession_sched_1(charge):
+            charge.expungement_result.set_type_eligibility(True)
+            charge.expungement_result.set_reason('Eligible under 137.225(5)(C)')
+
+        elif charge.level == 'Felony Class C' or 'Misdemeanor' in charge.level:
+            charge.expungement_result.set_type_eligibility(True)
+            charge.expungement_result.set_reason('Eligible under 137.225(5)(b)')
+
+        elif charge.level == 'Felony Class B':
+            self.class_b_felonies.append(charge)
+            charge.expungement_result.set_reason('Further Analysis Needed')
+
+        else:
+            charge.expungement_result.set_reason('Examine')
+
+    @staticmethod
+    def __ineligible_under_137_225_5(charge):
```

Comment:
There are a few methods that take a `charge` object as a parameter. These would be better placed in the charge class.